docker/auth.go

```diff
@@ -89,6 +90,12 @@ type ImageRepoDetails struct {
 
 	// ServerAddress is the auth server address.
 	ServerAddress string `json:"serveraddress,omitempty" yaml:"serveraddress,omitempty"`
+
+	// Region is the cloud region.
+	Region string `json:"region,omitempty" yaml:"region,omitempty"`
+
+	// ExpiresAt is the unix time in seconds and milliseconds when the authorization token expires.
+	ExpiresAt *time.Time `json:"expires-at,omitempty" yaml:"expires-at,omitempty"`
```

Comment:
Should these be in TokenAuthConfig struct?
Seems wrong to expose them here. It seems like they are related just to how token auth is done.

Reply:
Moved ExpiresAt to the token itself.